doc/examples/config/sharding.yaml

```diff
@@ -0,0 +1,39 @@
+credentials:
+  users:
+    guest:
+      roles: [super]
+
+iproto:
+  listen: 'unix/:./{{ instance_name }}.iproto'
+  advertise:
+    sharding: 'guest@'
+
+sharding:
+  bucket_count: 10000
+
+groups:
+  storages:
+    sharding:
+      roles: [storage]
+    replicasets:
+      storages-001:
+        instances:
+          storage-001:
+            database:
+              mode: rw
+          storage-002: {}
+      storages-002:
+        instances:
+          storage-003: {}
+          storage-004:
+            database:
+              mode: rw
+  routers:
+    replicasets:
+      routers-001:
+        sharding:
+          roles: [router]
+        instances:
+          router-001:
+            database:
+              mode: rw
```

Comment:
It is not necessary for an instance that is the only one in its replicaset. It'll be rw by default.